apps/ecs/fields/index.js

```diff
@@ -127,6 +141,11 @@ module.exports = {
       }
     ]
   },
+  'arc-number': {
+    isPageHeading: 'true',
+    className: ['govuk-input', 'govuk-!-width-two-thirds'],
```

Comment:
This field is looks little smaller in figma. so this classname - 'govuk-!-width-two-thirds' can be used.